Dockerfile

```diff
@@ -26,6 +26,10 @@ RUN apt-get install -y git unzip wget sudo curl build-essential gettext \
 RUN curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash - && \
     apt-get -y install nodejs libxcb-dri3-0 libxss1 libdrm2 libgbm1
 
+# install the redis server, bind to ipv4 (or it will not start)
+RUN apt-get -y install redis-server && \
+    sed -i "s/^bind 127.0.0.1.*/bind 127.0.0.1/g" /etc/redis/redis.conf
```

Comment:
https://github.com/redis/redis/issues/5055

Om een of andere reden is de default regel in redis.conf met de ipv6 en wenst dit in deze docker container niet te starten. Bij configuratie met alleen ipv4, werkt het wel.